### ISSUE TITLE
Controls Disappear on Moderately Sized Screen

### ISSUE THREAD

**Issue by mtekman** [schwartzlab-methods/too-many-cells-interactive#13]
https://github.com/schwartzlab-methods/too-many-cells-interactive/blob/3cf67fadc025694af5c9d54d5f26bb0d5668cc9b/react/src/Components/Layout.tsx#L17

The jump from a 1237x1000px screen to a 1238x1000px screen is extremely jarring, and needless since the screen renders the controls fine if I override this limit.

If the idea is to accomodate vertical screens such as phones, then maybe test for touch devices instead of purely orientation.
Or, let the flexbox of the controls flow into a 1 column system for narrow displays

When width is 900px wide for example,  the UI just looks too strange to be assigned a vertical orientation.

Some pseudocode to show hopefully better what I mean:

```
cases:
       if phone and vertical:  vertical display (two column controls)
       if phone and horizontal: horizontal display (one column controls)
       if not phone and 0 < width < 700:  vertical display (one column controls)
       if not phone and 700 < width < 900: horizontal display (one column controls)
       if anything else: horizontal display (two column controls)
```

**Comment by cklamann** [comment]
Hi @mtekman, thanks for submitting this issue and providing the examples and pseudocode.

You're certainly right that the UI needs work at the resolutions you indicate and your suggestions are helpful. I should have time to take a closer look later this week with updates to follow.

**Comment by cklamann** [comment]
Closed by #14
